packages/flutter/pubspec.yaml

```diff
@@ -25,31 +25,31 @@ dependencies:
   flutter:
     sdk: flutter
 
-  parse_server_sdk: ^5.1.3
+  parse_server_sdk: ^6.3.0
```

Comment:
This looks odd; it's incrementing by a major version; is this a breaking change?

Reply:
No, it's just updated and not a breaking change

Reply:
The reason for using this dependency is to use `parse_server_sdk` in `parse_server_sdk_flutter` and the reason for upgrading is to use the latest version of `parse_server_sdk`.

Reply:
OK, so upgrading parse_server_sdk to a higher major version is only then no breaking change for this SDK, if the upgrade doesn't cause any changes for the developer in this SDK's APIs; and if it isn't a transient dependency a developer can use, which could also require code changes for the developer.

Reply:
According to [this](https://github.com/parse-community/Parse-SDK-Flutter/releases/tag/dart-6.0.0) and review, I think breaking change will happen, maybe it is better not to upgrade this section, what do you think?

Reply:
Not sure, what do you suggest would be best?

Reply:
We have two ways

1. Bump the Flutter version (breaking change)
2. Return this part to its previous state

My suggestion is to return to the previous state and upgrade with a big feature in the future

Reply:
Sure, we'll go with your decision.

Reply:
Reverted to previous version.